src/pyedb/dotnet/edb_core/stackup.py

```diff
@@ -2187,14 +2185,18 @@ def _export_xml(self, file_path):
         return True
 
     @pyedb_function_handler()
-    def load(self, file_path):
+    def load(self, file_path, mode=0):
         """Import stackup from a file. The file format can be XML, CSV, or JSON.
 
 
         Parameters
         ----------
         file_path : str
             Path to stackup file.
+        mode : int
+            mode=`0` layer in layout not found in the stackup file are deleted and, mode=`1` if the number of layer
+            in the stackup file equals the number of stackup layer in the layout, layers are renamed according the the
+            file. Note that layer order matters, and has to be writtent from top to bottom layer in the file.
```

Comment:
```suggestion
        rename : bool
            If rename is ``False`` then layer in layout not found in the stackup file are deleted. Otherwise, if the number of layer
            in the stackup file equals the number of stackup layer in the layout, layers are renamed according the the
            file. Note that layer order matters, and has to be writtent from top to bottom layer in the file.
```